tests/test_cli.py

```diff
@@ -84,28 +84,14 @@ def test_parse_argv_no_options(tmpdir, option):
 
 
 @pytest.mark.parametrize('verbose', (True, False))
-def test_log_verbosity(runner, verbose, recipe_arguments):
+def test_log_verbosity(root_logger, runner, verbose, recipe_arguments):
     """Ensure that the verbosity is set properly on."""
-    options = ['--verbose'] if verbose else []
-    ctx = run.make_context('test_log_verbosity',
-                           options + recipe_arguments)
-    assert ctx.params['verbose'] is verbose
-
-    log = logging.getLogger()
-    # Initial state – if the test fails here, the app has changed
-    # and the test needs to be adjusted.
-    # The default log level is WARNING.
-    # https://docs.python.org/3/library/logging.html
-    assert log.getEffectiveLevel() == logging.WARNING
```

Comment:
As the callback is run every time, there is no good way reset the initial state. Dropping this check should not harm anything.